web/translation/translate.en_US.toml

```diff
@@ -373,7 +373,7 @@
 "NetflixWARP" = "Netflix"
 "NetflixWARPDesc" = "Routes traffic to Netflix via WARP."
 "MetaWARP" = "Meta"
-"MetaWARPDesc" = "Routes traffic to Meta (Instagram, Facebook, WhatsApp, Threads,...) via WARP."
+"MetaWARPDesc" = "Routes traffic to Meta (Instagram, Facebook, WhatsApp,...) via WARP."
```

Comment:
Why you removing it ?

Reply:
It is, but I expect issues and questions from users about if it supports `Threads` as well.
Also assume that some people add `Threads` as separated domain